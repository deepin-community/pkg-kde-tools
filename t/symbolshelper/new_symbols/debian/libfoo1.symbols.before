# SymbolsHelper-Confirmed: 0.1 amd64 i386
libfoo.so.1 libfoo1 #MINVER#
 (subst)_Z3foo{size_t} 0.1
