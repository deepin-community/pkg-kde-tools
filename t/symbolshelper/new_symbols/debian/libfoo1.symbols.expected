# SymbolsHelper-Confirmed: 1.0 amd64 i386
libfoo.so.1 libfoo1 #MINVER#
 (subst)_Z3bar{size_t} 1.0
 _Z3bazj 1.0
 (subst)_Z3foo{size_t} 0.1
